src/cpg_flow/targets/pedigreeinfo.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
```

Comment:
Could we use snakecase for the naming of these files in line with our other repos?